ansible/roles/apache/tasks/main.yml

```diff
@@ -13,6 +13,19 @@
     - expires
     - filter
 
+#
+# Preventing "Apache is running a threaded MPM, but your PHP Module..."
+# http://stackoverflow.com/questions/19185268/php-and-apache-thread-safe-error
+- name: Disabling mpm_event module
+  sudo: yes
+  command: a2dismod mpm_event
+
+- name: Enabling mpm_prefork module
+  sudo: yes
+  command: a2enmod mpm_prefork
+  notify: restart apache
+#
+
```

Comment:
Needed to add these module updates to the apache2 set up in order to prevent this error:

<img width="929" alt="screenshot 2016-01-30 00 10 41" src="https://cloud.githubusercontent.com/assets/1557348/12697750/d7473c02-c758-11e5-949d-c3416f7ab52f.png">